src/main/templates/main/about.html

```diff
@@ -0,0 +1,3 @@
+{% extends 'main/base.html' %}
+{% block title %}About us{% endblock %}
+{% block body %} <h5>Information about store</h5> {% endblock %}
```

Comment:
new line.

Reply:
and probably it os a matter of taste but I think that it is better to structure such files like this:
```suggestion
{% block body %} 
  <h5>Information about store</h5> 
{% endblock %}
```